hera_sim/defaults.py

```diff
@@ -274,8 +274,8 @@ def _check_config(self):
         """Check and warn if any keys in the configuration are repeated."""
         # initialize dictionaries that enumerate the key, value pairs
         # in the raw configuration dictionary
-        counts = {key: 0 for key in self().keys()}
-        values = {key: [] for key in self().keys()}
+        counts = dict.fromkeys(self().keys(), 0)
+        values = dict.fromkeys(self().keys(), [])
```

Comment:
This is a bad idea. Using `dict.fromkeys(blah, mutable)` creates pointers to only one instance of `mutable`. For example, try running the following code:
```
mydict = dict.fromkeys([1,2,3], [])
mydict[1].append(3)
mydict[2].append(5)
print(mydict[3])
```

This is *not* an issue when using a dict comprehension (hence why it was done that way initially).

Reply:
Ah! I didn't realize this! Thanks for catching what would have been an awful bug.